xds/internal/balancer/priority/balancer_child.go

```diff
@@ -0,0 +1,95 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package priority
+
+import (
+	"google.golang.org/grpc/balancer"
+	"google.golang.org/grpc/balancer/base"
+	"google.golang.org/grpc/connectivity"
+	"google.golang.org/grpc/resolver"
+	"google.golang.org/grpc/serviceconfig"
+)
+
+type childBalancer struct {
+	name   string
+	parent *priorityBalancer
```

Comment:
The only reason `parent` seems to be used here is to access the `balancerGroup`. Could we instead have the operations on the `balancerGroup` handled directly by the `priorityBalancer`. If that would make things complex, at least we can directly accept the `balancerGroup` here instead of the `parent`. What do you think?

Reply:
I thought the children also need the logger from parent. But it doesn't seem to call it.
I'm keeping it in case we need info logs later